src/rpc/handlers/AccountCurrencies.h

```diff
@@ -27,6 +27,11 @@
 #include <set>
 
 namespace RPC {
+/**
```

Comment:
Let's add a space line to each of these comments if it's missing - to match the format everywhere.
Also, I think the last link should be `For more details see https://...html` and be separated by a line of space from the brief.